sdk/common/src/test/java/io/opentelemetry/sdk/common/InstrumentationLibraryInfoTest.java

```diff
@@ -10,18 +10,21 @@
 
 import org.junit.jupiter.api.Test;
 
-/** Tests for {@link InstrumentationLibraryInfo}. */
+/** Tests for {@link io.opentelemetry.sdk.common.InstrumentationLibraryInfo}. */
```

Comment:
Looks like some qualified imports got automatically added by intellij or something

Reply:
You have to qualify the imports, even in comments, or else you get an error for using deprecated classes.

Reply:
Ah thought if it's in the same package that wouldn't be an issue but maybe not